txtorcon/torcontrolprotocol.py

```diff
@@ -356,11 +357,19 @@ def get_info(self, *args):
 
         :return:
             a ``Deferred`` which will callback with a dict containing
-            the keys you asked for. This just inserts ``parse_keywords``
-            in the callback chain; if you want to avoid the parsing
+            the keys you asked for. If you want to avoid the parsing
             into a dict, you can use get_info_raw instead.
         """
-        return self.get_info_raw(*args).addCallback(parse_keywords)
+        lines = yield self.get_info_raw(*args)
+        rtn = {}
+        key = None
+        for line in lines.split('\n'):
+            if line.split('=', 1)[0] in args:
+                key = line.split('=', 1)[0]
+                rtn[key] = line.split('=', 1)[1]
```

Comment:
For a multiline response, we do not have a trailing newline in the end. We can see this from the test, https://github.com/meejah/txtorcon/blob/master/test/test_torcontrolprotocol.py#L430